test/enumcustomname/enumcustomname.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+package enumcustomname;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+enum MyEnum {
+	option (gogoproto.goproto_enum_stringer) = true; // ensure it behaves correctly with stringer.
```

Comment:
I'll move that down to the disabled prefix test, since we really want to make sure the stringer is being generated with the overrides.